build.zig

```diff
@@ -73,19 +86,24 @@ pub fn build(b: *std.build.Builder) !void {
     // Add tests.
     const test_step = b.step("test", "Run tests");
     for ([_][]const u8{ "main", "wav", "file", "gen" }) |module| {
+        const test_name = try std.fmt.allocPrint(allocator, "{s}-tests", .{module});
+        defer allocator.free(test_name);
         const test_module = try std.fmt.allocPrint(allocator, "src/{s}.zig", .{module});
```

Comment:
why not `b.fmt(...)`? No need allocator!

Reply:
applied to main in ef79489af42737b6d3e8d6d72b5de514da9f38c9